extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4593,6 +4659,7 @@ var globalRequire = require;
 						case eventType.USER_CLOSE:
 						case eventType.SKIPPED:
 						case eventType.COMPLETE:
+							this.adPlaybackStarted = false;
 							if (this.videoControllerWrapper && (ad && ad.isLinear()))
 							{
 								_stopTimeUpdater();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_stopTimeUpdater" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)